### DISCUSSION TITLE
Anyway to suppress the command prompt while auto start is enabled?

### DISCUSSION THREAD

**Discussion by earl088** [Rem0o/FanControl.Releases#734] [Q&A]
This gave my quite a scare and though my computer had a form of malware on it and I would love to be able to still autostart FanControl but I want to get rid of the command prompt pop-up for various reasons.

**Comment by Rem0o** [comment]
For various reasons, the command prompt starting FanControl is more reliable and works for a greater ammount of people.  You can always edit the scheduled task.

**Reply by earl088** [reply]
Edit it so that it will not show the command prompt?

**Comment by xhul-dev** [comment]
In the options, try setting "Delay" to "1 sec".
If your machine & system are fast enough, there's a high chance the cmd window will actually pop before your monitor even displays the desktop, i've never seen it once by doing that.

**Reply by earl088** [reply]
I tried it and I can still see it come up. 12900K and SN850 still not fast enough :'(

**Reply by xhul-dev** [reply]
> I tried it and I can still see it come up. 12900K and SN850 still not fast enough :'(

Your monitor is probably too fast then.

You should definitely try what Rem0o suggested:
In the task scheduler, double-click on the task called "FanControl".
Then, in the "Actions" tab, double-click on the action called "Start a program".
Change the "Program/script" field to point to FanControl.exe directly, by using the "Browse..." button.
Also, remove everything that's in the "Add arguments (optional)" field.
Save changes.
Finally, restart (not shutdown) Windows to be sure that Fan Control still launches properly on Windows startup.

**Reply by earl088** [reply]
Not sure what you mean that the monitor is too fast, the monitor turns on immediately as the POST starts and does not go off anymore. I wanted to follow what Rem0o suggested but he did not provide any steps. Thank you for providing the steps.

**Reply by xhul-dev** [reply]
> Not sure what you mean that the monitor is too fast, the monitor turns on immediately as the POST starts and does not go off anymore. I wanted to follow what Rem0o suggested but he did not provide any steps. Thank you for providing the steps.

For some monitors, the resolution switch from POST to Windows sometimes takes enough time that you get a black screen while the desktop initialises (in my case, the cmd window actually happens before the desktop shows).
That's what i meant =]

**Comment by earl088** [comment]
Sorry for the thread necro! I recently did a clean install and just like in the past and up until today I still dislike the command prompt showing up for 0.1seconds and I tried the alternative steps to get fanspeed running and it was all golden up until this clean install. For some reason unknown to me it no longer successfully launches instead which is sad. Any other work arounds I could try?
